### DISCUSSION TITLE
Qdrant result score above 1 after indexing with small dataset

### DISCUSSION THREAD

**Discussion by ajainvivek** [qdrant/qdrant#2660] [Q&A]
It appears that the results from a search using Cosine similarity in the qdrant system have scores exceeding 1. I've noticed scores like 500 and 724. This increase in scores seems to occur after the vectors have been indexed. Can someone provide assistance or insights into this issue?

**Comment by generall** [comment]
it looks like you are using binary quantization. BQ requires enabling a re-scoring option to provide correct score values
